internal/pkg/vm/internal/vmcontext/invocation_context.go

```diff
@@ -424,6 +424,10 @@ func (ctx *invocationContext) Send(toAddr address.Address, methodNum abi.MethodN
 	from := ctx.msg.to
 	fromActor := ctx.toActor
 
+	if fromActor.Balance.LessThan(value) {
```

Comment:
found while writing: https://github.com/filecoin-project/chain-validation/pull/191/files#diff-834339dac0ac5f1e95be0521aaf42147R323.

If the fromActor's balance is not checked here the `transfer` call in `newCtx.invoke()` will panic.

Reply:
Thanks. Moved to https://github.com/filecoin-project/go-filecoin/pull/4117